src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -28,6 +19,14 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using ProtoCore.Namespace;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml;
```

Comment:
why do you need to reference system xml?

Reply:
whoops - my fault, did not see these were already there - @QilongTang - I can't remember how are these supposed to be sorted?

Reply:
Systems ones after project specific, scope from specific to generic